app/views/user_annotations/_form.html.erb

```diff
@@ -0,0 +1,79 @@
+<%= form_for(@user_annotation) do |f| %>
+  <% if @user_annotation.errors.any? %>
+		<div class="bs-callout bs-callout-danger">
+			<h4><%= pluralize(@user_annotation.errors.count, 'error') %> prohibited this study from being saved:</h4>
+			<ul>
+				<% @user_annotation.errors.full_messages.each do |message| %>
+					<li><%= message %></li>
+				<% end %>
+			</ul>
+		</div>
+	<% end %>
+  <div class="form-group row">
+    <div class="col-sm-3">
+      <%= f.label 'Name:' %>
+      <%= f.text_field :name, class: 'form-control need-text', autocomplete: 'off' %>
+    </div>
+    <div class="col-sm-3">
+      <%= f.label 'Cluster:' %>
```

Comment:
While this works to disable the field to prevent an error, really what you should do is follow the paradigm you set below of creating an anonymous input and setting the value rather than putting an incorrect value inside the _id association fields.